view/frontend/web/template/payment/offsite-installment-form.html

```diff
@@ -18,10 +18,10 @@
                 </div>
             </div>
         </div>
-        <div data-bind="visible: orderValueTooLow()" class="page messages">
+        <div data-bind="visible: orderValueTooLow() && isActive()" class="page messages">
```

Comment:
Just a minor thing, maybe `isActive() && orderValueTooLow()` would be more familiar to our eyes.
_just personal opinion, feel free to ignore if you don't want to change_

Reply:
ok.. it is more logical to check first if it is active at all.